profiler/profiler.go

```diff
@@ -240,8 +240,7 @@ func (p *profiler) collect(ticker <-chan time.Time) {
 					profs, err := p.runProfile(t)
 					if err != nil {
 						log.Error("Error getting %s profile: %v; skipping.", t, err)
-						tags := make([]string, len(p.cfg.tags), len(p.cfg.tags)+1)
-						copy(tags, p.cfg.tags)
+						tags := append(p.cfg.tags.Get(), t.Tag())
```

Comment:
🐞  This adds `t.Tag()` twice, see line below.

Reply:
Fixed